domains/job.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fweeaaaa1",
+        "email": "gorgonfweman@outlook.com"
+    },
+    "record": {
+        "A": "128.204.223.115"
```

also A record should be an array
```suggestion
        "A": ["128.204.223.115"]
```